lib/elixir/lib/inspect.ex

```diff
@@ -478,6 +481,15 @@ defimpl Inspect, for: Any do
     end
   end
 
+  defp validate_option(option_list, fields, module, caller) do
+    if not Enum.empty?(option_list -- fields) do
+      description = "When deriving inspect protocol of #{module}, values must match struct fields"
```

Comment:
```suggestion
      description = "unknown fields #{inspect(...)} given when deriving the Inspect protocol for #{inspect(module)}. :only and :except values must match struct fields"
```

Reply:
Originally i had something like this, but the problem is that `inspect/1` is not yet available. I looked at some of the erlang `:io_lib` libraries and at `:erlang.display/1` but those weren't exactly right. I'll take another stab at it using `Enum.join/2` which will trim off the `:`s

Reply:
Oh, we unimport inspect to avoid conflcit. You can use `Kernel.inspect(module)` instead. :)

Reply:
> Oh, we unimport inspect to avoid conflcit. You can use `Kernel.inspect(module)` instead. :)

Thanks! it's wild how tricky it gets when i think I'm missing one function :) 
updated: dc203bfe0c8d9f9999f59b0897056f4e2ccccb2a